src/components/soknadpanel/SporsmalVarianter/Dato.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { BodyShort, Heading } from '@navikt/ds-react';
+
+import { cleanId } from '../../../utils/stringUtils';
+import { formatDate } from '../../../utils/dateUtils';
+
+import { SporsmalVarianterProps } from './SporsmalVarianter';
+import SporsmalListItem from './shared/SporsmalListItem';
+import SporsmalList from './shared/SporsmalList';
+import SporsmalListItemNested from './shared/SporsmalListItemNested';
+
+function Dato({ sporsmal }: SporsmalVarianterProps): JSX.Element | null {
+    const listItemId = cleanId(sporsmal.sporsmalstekst);
+
+    if (!sporsmal.svar || sporsmal.svar.length === 0) return null;
+
+    return (
+        <SporsmalListItem listItemId={listItemId}>
+            <Heading id={listItemId} size="xsmall" level="4">
+                {sporsmal.sporsmalstekst}
+            </Heading>
+            <SporsmalList>
+                {sporsmal.svar.map((svar, index) => {
+                    return (
+                        <SporsmalListItemNested listItemId={listItemId + index} key={listItemId + index}>
+                            <BodyShort size="small">{svar?.verdi && formatDate(svar.verdi)}</BodyShort>
+                        </SporsmalListItemNested>
+                    );
+                })}
```

Comment:
Hmm, dersom svar her er nullable, burde man kanskje gjort noe ala sånn?

```suggestion
                {sporsmal.svar.filter(notNull).map((svar) => {
                    const svarId = cleanId(svar);
                    return (
                        <SporsmalListItemNested listItemId={svarId} key={svarId}>
                            <BodyShort size="small">{svar.verdi && formatDate(svar.verdi)}</BodyShort>
                        </SporsmalListItemNested>
                    );
                })}
```